packages/core/tocco-ui/src/Icon/FontAwesomeAdapter.js

```diff
@@ -237,13 +242,20 @@ import {StyledFontAwesomeAdapterWrapper} from './StyledComponents'
  * The following icons are solely used to support custom display expressions and should not be removed: faSquareSolid
  */
 
-const FontAwesomeAdapter = ({icon, style, hasFixedWidth}) => {
+const FontAwesomeAdapter = props => {
+  const {icon, style, hasFixedWidth} = props
   const getIcon = icon.includes(',') ? icon.replace(/\s+/, '').split(',') : icon
   const getIconColor = {...(_get(style, 'color') && {color: style.color})}
 
   return (
     <StyledFontAwesomeAdapterWrapper>
-      <FontAwesomeIcon icon={getIcon} style={style} {...getIconColor} fixedWidth={hasFixedWidth} />
+      <FontAwesomeIcon
+        icon={getIcon}
+        style={style}
+        {...getIconColor}
+        fixedWidth={hasFixedWidth}
+        {..._omit(props, ['icon', 'style', 'color', 'fixedWidth'])}
```

Comment:
suggestion (non-blocking): you can also destruct the remaining keys instead of remove destructed keys from props

```
const {icon, style, hasFixedWidth, ...otherProps} = props

...

<FontAweseomIcon icon={...} {...otherProps} />
```